test/v3/fixtures/retrieveBalanceAndTransactions/DocumentDetailsFixture.scala

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v3.fixtures.retrieveBalanceAndTransactions
+
+import play.api.libs.json._
+import v3.models.response.retrieveBalanceAndTransactions.{DocumentDetails, LastClearing, LatePaymentInterest, ReducedCharge}
+
+object DocumentDetailsFixture {
+
+  // MTD model:
+
+  val lastClearing: LastClearing = LastClearing(
+    lastClearingDate = Some("2018-04-05"),
+    lastClearingReason = Some("Incoming Payment"),
+    lastClearedAmount = Some(3.99)
+  )
+
+  val latePaymentInterest: LatePaymentInterest = LatePaymentInterest(
+    latePaymentInterestId = Some("1234567890123456"),
+    accruingInterestAmount = Some(4.99),
+    interestRate = Some(6.99),
+    interestStartDate = Some("2020-04-01"),
+    interestEndDate = Some("2020-04-05"),
+    interestAmount = Some(8.99),
+    interestDunningLockAmount = Some(7.99),
+    interestOutstandingAmount = Some(9.99)
+  )
+
+  val reducedCharge: ReducedCharge = ReducedCharge(
+    chargeType = Some("???"),
+    documentNumber = Some("???"),
+    amendmentDate = Some("2018-04-05"),
+    taxYear = Some("2017-18")
+  )
+
+  val documentDetails: DocumentDetails = DocumentDetails(
+    taxYear = Some("2020-21"),
+    documentId = "1455",
+    formBundleNumber = Some("88888888"),
+    creditReason = Some("Voluntary Payment"),
+    documentDate = "2018-04-05",
+    documentText = Some("ITSA- Bal Charge"),
+    documentDueDate = Some("2021-04-05"),
+    documentDescription = Some("ITSA- POA 1"),
+    originalAmount = 1.99,
+    outstandingAmount = 2.99,
+    lastClearing = Some(lastClearing),
+    isChargeEstimate = true,
+    isCodedOut = true,
+    paymentLot = Some("AB1023456789"),
+    paymentLotItem = Some("000001"),
+    effectiveDateOfPayment = Some("2021-04-05"),
+    latePaymentInterest = Some(latePaymentInterest),
+    amountCodedOut = Some(5.99),
+    reducedCharge = Some(reducedCharge)
+  )
+
+  val documentDetailsWithoutDocDueDate: DocumentDetails = documentDetails.copy(documentDueDate = None)
+
+  val documentDetailsMinimal: DocumentDetails = DocumentDetails(
+    None,
+    documentId = "1455",
+    None,
+    None,
+    documentDate = "2018-04-05",
+    None,
+    documentDueDate = Some("2021-04-05"),
```

Comment:
As this is `documentDetailsMinimal` it would make more sense if `documentDueDate` is `None`?